app/services/container_dashboard_service.rb

```diff
@@ -95,6 +96,31 @@ def providers
     result.values
   end
 
+  def alerts
+    if @ems.present?
+      warnings = @ems.miq_alert_statuses.where(:severity => "warning").count
+      errors = @ems.miq_alert_statuses.where(:severity => "error").count
+    else
+      warnings = MiqAlertStatus.where(:severity => "warning").count
+      errors = MiqAlertStatus.where(:severity => "error").count
```

Comment:
Are alert statuses exclusive to containers providers? Seems like there should be a filter here